debezium-core/src/main/java/io/debezium/transforms/Filter.java

```diff
@@ -178,31 +120,32 @@ public static NullHandling parse(String value, String defaultValue) {
     private Engine engine;
     private NullHandling nullHandling;
 
-    public Filter() {
-    }
-
     @Override
     public void configure(Map<String, ?> configs) {
         final Configuration config = Configuration.from(configs);
-        final ExpressionLanguage language = ExpressionLanguage.parse(config.getString(LANGUAGE));
-        final String expression = config.getString(EXPRESSION);
-        if (language == null || expression == null) {
-            throw new DebeziumException(
-                    "Options '" + LANGUAGE + "' and '" + EXPRESSION + "' must be present. Language must be one of (" + Arrays.toString(ExpressionLanguage.values())
-                            + ")");
-        }
-        LOGGER.info("Using language '{}' to evaluate expression '{}'", language.getValue(), expression);
+        final String expression = config.getStringNonEmpty(EXPRESSION);
```

Comment:
As enum is no longer used would it it be better to switch to the `withValidator` on config definition?

Reply:
Good idea in general; just tried it, but it seems validation logic isn't invoked (expected `DebeziumException` isn't raised). Is this supported for SMT options actually?

Reply:
@gunnarmorling IMHO you need to call config.validate(), see `ExtractNewRecord` for example

Reply:
Gotcha, pushed an update for that.

Reply:
@gunnarmorling Looks like there is a formatting issue